internal/config/config.go

```diff
@@ -251,6 +251,9 @@ func parsePeer(p peer) (*Peer, error) {
 	if ip == nil {
 		return nil, fmt.Errorf("invalid peer IP %q", p.Addr)
 	}
+	if strings.ContainsAny(p.Addr, ":") == true {
```

Comment:
I wouldn't put this change here. Instead, look for the call to `bgpNew` in speaker/bgp_controller.go . There is a `Sprintf` call there that builds the address, and it should be using `net.JoinHostPort` instead. That will make the stdlib add bracket as needed, depending on the address family etc.

Reply:
Ok thanks for the tips, I will close this PR and take a time for check theses functions. 

:+1: